test/test_examples.py

```diff
@@ -747,7 +753,7 @@ def insert_docs():
                 db.inventory.insert_one({"username": "alice"})
                 db.inventory.delete_one({"username": "alice"})
```

Comment:
We should probably put a short sleep in here (5ms maybe) to avoid a tight loop here.